lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -66,7 +68,7 @@
         % endif
       >
 <article class="course${mode_class}" aria-labelledby="course-title-${enrollment.course_id}" id="course-card-${course_card_index}">
-  <% course_target = reverse(course_home_url_name(course_overview.id), args=[six.text_type(course_overview.id)]) %>
+  <% course_target = reverse(course_home_url_name(course_overview.id), args=[six.text_type(course_overview.id)]) if course_home_legacy_is_active(course_overview.id) else get_learning_mfe_home_url(course_key=course_overview.id, view_name="home") %>
```

Comment:
Minor thing, but while you're in here, do you mind switching the use of `six.text_type(course_overview.id)` -> `str(course_overview.id)`

Happy to give more context here if you haven't interacted with six before